nav.css

```diff
@@ -104,18 +104,42 @@
 	padding-left: var(--ncn-pad-h);
 	padding-right: var(--ncn-pad-h);
 }
+:where(.ncn-home-logo-full) {
+	--_ncn-home-logo-lines-color: var(--ncn-home-logo-lines-color, #05bdba);
+	--_ncn-home-logo-text-color: var(--ncn-home-logo-text-color, #014847);
```

Comment:
variable fallbacks should be what is set to the variable instead of an explicit value (In 108, it should be `var(--ncn-color-teal)` instead of `#05bdba`)